gengokit/gentesthelper/helper_test.go

```diff
@@ -1,9 +1,39 @@
 package gentesthelper
 
 import (
+	"strings"
 	"testing"
 )
 
 func TestDiffGoCode(t *testing.T) {
+	code := []string{
+		// Normal
+		`func Foo() {
+			println("test")
+		}`,
+		// Trailing whitespace on both ends
```

Comment:
Nit: it's only trailing after the code. It's "surrounded blank lines," to be specific. Really really minor, but I did pause for a second trying to figure out what it meant before I realized it meant something different than it actually said.